sympy/simplify/trigsimp.py

```diff
@@ -426,12 +426,52 @@ def analyse_gens(gens, hints):
 _trigs = (TrigonometricFunction, HyperbolicFunction)
 
 
-def trigsimp(expr, **opts):
+def _trigsimp_inverse(rv):
+
+    def check_args(x, y):
+        try:
+            return x.args[0] == y.args[0]
+        except IndexError:
+            return False
+
+    def f(rv):
+        if type(rv) is atan2:
+            y, x = rv.args
+            if y.is_Mul and y.args[0] == -1:
```

Comment:
```python
>>> from sympy.core.function import _coeff_isneg
>>> _coeff_isneg(-x-3)
False
>>> _coeff_isneg(-x)
True
>>> _coeff_isneg(x)
False
```

Also, I think you can use `if isinstance(rv, atan2)`, etc...

Reply:
Updated!